pre_commit/commands/hook_impl.py

```diff
@@ -76,6 +76,7 @@ def _ns(
         remote_url: Optional[str] = None,
         commit_msg_filename: Optional[str] = None,
         checkout_type: Optional[str] = None,
+        is_squash_commit: Optional[str] = None,
```

Comment:
I see this passed along, is this exposed to hooks in any way?  I think the current examples for other hooks expose these as environment variables -- probably something like `PRE_COMMIT_IS_SQUASH_COMMIT`?

Reply:
An environmental variable is added, as is a `precommit run` flag. This breaks some tests